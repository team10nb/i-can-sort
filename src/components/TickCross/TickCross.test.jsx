/*
    Author: Shiliang Chen, Yijie Lu
*/

import { render } from "@testing-library/react";

test( 'should ', () => {
    expect(true).toBeTruthy();
})
