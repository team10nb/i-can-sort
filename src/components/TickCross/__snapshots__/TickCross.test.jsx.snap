// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should match TickCross snapshot 1`] = `
Object {
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <svg
        class="progress-icon"
        height="22px"
        viewBox="0 -5 50 50"
        width="18px"
      >
        <div>
          mockModuleProgress
        </div>
        <div>
          mockModuleProgress
        </div>
      </svg>
    </div>
  </body>,
  "container": <div>
    <svg
      class="progress-icon"
      height="22px"
      viewBox="0 -5 50 50"
      width="18px"
    >
      <div>
        mockModuleProgress
      </div>
      <div>
        mockModuleProgress
      </div>
    </svg>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
